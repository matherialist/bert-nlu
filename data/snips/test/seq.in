can you turn the lights on please
no fishing
we'll never find a way
can you switch off the light please
switch the light on please
we'll be there
what a big dog that is
could you switch the lighting on please
i want chocolate
could you turn on the light
activate light please
could you please turn the lighting on
turn on ilumination please
what about turning the lighting on
please activate ilumination device
he catches the train every morning
was the building on fire
deactivate the ilumination device please
hold on i need at least a few minutes
please switch on ilumination
i've got a rockin pneumonia
can you turn on the lighting please
could you switch the light off please
could you give me the light please
can you iluminate the room please
enable the ilumination please
my dog jake barked at the mail carrier
every decision has an antecedent
go to old chinatown
what about switching ilumination off
can you turn the lights off please
with just a wave of a strong right hand
freedom yeah
can you switch on the ilumination please
just a minor operation
can you please disable light device
come back for more eh
can you switch off the lighting please
freedom so i can give
can you please turn ilumination off
i'm gonna write a little letter
can you please enable the ilumination
with just a little bit of
please turn off the lights
can you light the room up
of what he's going to see
could you give me light please
activate the ilumination device
can you please switch on the lights
please light the room up
why you scarred him for life
please enable ilumination
could you switch the ilumination on please
manic depression is catching my soul
please turn the ilumination on
could you switch on the light please
are there not flying saucers or ufos
disable the light
disable ilumination device
letâ€™s go i'm sick of waiting
where troubles melt like lemon drops
could you turn off the lights please
you've been running and hiding much too long
can you turn the light on please
can you please enable lighting device
can you disable lighting
what about turning the light off
i have some announcements to make
make you happy as a clown
pointing their plastic finger at me
i know it's everybody sin
please disable lighting device
disable lighting
it's very far away
what about turning the lighting off
please disable the light
could you switch the lighting off please
activate the ilumination
can you disable the ilumination
go get your lover
yeah lose some sleep
a crowd of people stood and stared
can you activate the ilumination
please activate the light
tom is a local boy who made it big
that's what i want now
switch on the lights
can you deactivate lighting device please
can you enable ilumination
what about turning the light on
turn the lights on please
to get there
you know it's true oh
oh that must be the babysitter
and battle grounds
i'm not going to copy you
could you turn off the light
can you turn the light off please
please enable ilumination device
she's got a crazy partner
could you switch on ilumination
that's a really big apple
i have as much respect
disable the ilumination
you don't have to
i am doing just wonderful
can you please enable light device
can you turn the lighting off please
please turn the light on
could you please turn off the lighting
could you turn the lighting on please
turn off the ilumination please
and it's just a little bit of
sittin down by the rhythm reviews
i dug up a grave
activate the lighting please
could you switch off the light
what a kick in the teeth
could you please switch off the lighting
turn ilumination off please
please switch off lighting
can you deactivate light
i'm sorry
turn the lights on
what about switching on ilumination
can you enable the ilumination please
help me please
good gracious ridiculous
could you switch on the lighting please
enable the light
when your time's on the door
your old man let you down
gonna mail it to my local dj
turn ilumination on please
go to the opera
darling won't you ease my worried mind
what about switching on the ilumination
with cotton candy
turn the ilumination off
endless talking
made a victim out of your life
enable lighting please
can you please disable lighting
can you please deactivate ilumination device
could you please switch on the lighting
can you activate the light device
can you please turn off the light
can you please activate the ilumination
please disable light device
i need ilumination
can you please deactivate the light
can you give me the light
a place behind the sun
how to go about getting it
worn like a mask of self hate
can you activate the lighting device please
it's a groovy name
can you please deactivate the ilumination
what about turning on ilumination
can you please switch on the ilumination
