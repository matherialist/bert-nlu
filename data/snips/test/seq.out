O O B-light_on O O I-light_on O
O O
O O O O O
O O B-light_off I-light_off O O O
B-light_on O O I-light_on O
O O O
O O O O O O
O O B-light_on O O I-light_on O
O O O
O O B-light_on I-light_on O O
B-light_on O O
O O O B-light_on O O I-light_on
B-light_on I-light_on O O
O O B-light_on O O I-light_on
O B-light_on O O
O O O O O O
O O O O O
B-light_off O O O O
O O O O O O O O O
O B-light_on I-light_on O
O O O O O
O O B-light_on I-light_on O O O
O O B-light_off O O I-light_off O
O O B-light_on O O I-light_on O
O O B-light_on O O O
B-light_on O O O
O O O O O O O O
O O O O O
O O O O
O O B-light_off O I-light_off
O O B-light_off O O I-light_off O
O O O O O O O O O
O O
O O B-light_on I-light_on O O O
O O O O
O O O B-light_off O O
O O O O O
O O B-light_off I-light_off O O O
O O O O O
O O O B-light_off O I-light_off
O O O O O O
O O O B-light_on O O
O O O O O O
O B-light_off I-light_off O O
O O B-light_on O O I-light_on
O O O O O O
O O B-light_on O I-light_on O
B-light_on O O O
O O O B-light_on I-light_on O O
O B-light_on O O I-light_on
O O O O O O
O B-light_on O
O O B-light_on O O I-light_on O
O O O O O O
O B-light_on O O I-light_on
O O B-light_on I-light_on O O O
O O O O O O O
B-light_off O O
B-light_off O O
O O O O O O
O O O O O O
O O B-light_off I-light_off O O O
O O O O O O O O
O O B-light_on O O I-light_on O
O O O B-light_on O O
O O B-light_off O
O O B-light_off O O I-light_off
O O O O O O
O O O O O O
O O O O O O
O O O O O
O B-light_off O O
B-light_off O
O O O O
O O B-light_off O O I-light_off
O B-light_off O O
O O B-light_off O O I-light_off O
B-light_on O O
O O B-light_off O O
O O O O
O O O O
O O O O O O O
O O B-light_on O O
O B-light_on O O
O O O O O O O O O
O O O O O
B-light_on I-light_on O O
O O B-light_off O O O
O O B-light_on O
O O B-light_on O O I-light_on
B-light_on O O I-light_on O
O O O
O O O O O
O O O O O O
O O O
O O O O O O
O O B-light_off I-light_off O O
O O B-light_off O O I-light_off O
O B-light_on O O
O O O O O
O O B-light_on I-light_on O
O O O O O
O O O O O
B-light_off O O
O O O O
O O O O O
O O O B-light_on O O
O O B-light_off O O I-light_off O
O B-light_on O O I-light_on
O O O B-light_off I-light_off O O
O O B-light_on O O I-light_on O
B-light_off I-light_off O O O
O O O O O O O
O O O O O O
O O O O O
B-light_on O O O
O O B-light_off I-light_off O O
O O O O O O
O O O B-light_off I-light_off O O
B-light_off O I-light_off O
O B-light_off I-light_off O
O O B-light_off O
O O
B-light_on O O I-light_on
O O B-light_on I-light_on O
O O B-light_on O O O
O O O
O O O
O O B-light_on I-light_on O O O
B-light_on O O
O O O O O O
O O O O O O
O O O O O O O
B-light_on O I-light_on O
O O O O
O O O O O O O
O O B-light_on I-light_on O O
O O O
B-light_off O O I-light_off
O O
O O O O O O O
B-light_on O O
O O O B-light_off O
O O O B-light_off O O
O O O B-light_on I-light_on O O
O O B-light_on O O O
O O O B-light_off I-light_off O O
O O O B-light_on O O
O B-light_off O O
O B-light_on I-light_on
O O O B-light_off O O
O O B-light_on O O I-light_on
O O O O O
O O O O O O
O O O O O O O
O O B-light_on O O O O
O O O O
O O O B-light_off O O
O O B-light_on I-light_on O
O O O B-light_on I-light_on O O
